### PR TITLE
Killed standalone LB

### PR DESCRIPTION
**Pull request by alena1108**

### CODE REVIEW

**Comment by alena1108** [comment]
Issues that will get fixed along:
https://github.com/rancher/rancher/issues/2047

**Comment by alena1108** [comment]
@ibuildthecloud could you please review. Don't get scared by file change number, its mostly code removal.

**Comment by ibuildthecloud** [comment]
rebase?

**Comment by ibuildthecloud** [comment]
OMG  +450 −8,467.  Best PR ever!!!!!!

**Comment by alena1108** [comment]
@ibuildthecloud I know, even my OCD inner self got quiet after the 100th removed file.

File: code/iaas/agent-instance/src/main/java/io/cattle/platform/agent/instance/factory/impl/AgentInstanceBuilderImpl.java [hunk 1]
```diff
@@ -50,7 +52,11 @@ public AgentInstanceBuilderImpl(AgentInstanceFactoryImpl factory, Instance insta
         this(factory);
         this.accountId = instance.getAccountId();
         this.zoneId = instance.getZoneId();
-        this.uri = "event:///instanceId=" + instance.getId();
+        String uriPrefix = "event";
+        if (StringUtils.equals(SystemContainer.LoadBalancerAgent.name(), instance.getSystemContainer())) {
```
Comment (ibuildthecloud) [inline code/iaas/agent-instance/src/main/java/io/cattle/platform/agent/instance/factory/impl/AgentInstanceBuilderImpl.java]: This is a hack.  Can you just pass in the uri or uri prefix somehow?

**Comment by ibuildthecloud** [comment]
@alena1108 We can't actually drop the tables in this release.  Then rollback isn't possible.  We can either just leave it as is for this release (drop the tables in a future release), or modify code/iaas/model/src/test/resources/codegen.xml to exclude the tables we don't use anymore.  I prefer the later.

File: code/iaas/agent-instance/src/main/java/io/cattle/platform/agent/instance/factory/impl/AgentInstanceBuilderImpl.java (hunk 1 above)
Comment (alena1108) [inline code/iaas/agent-instance/src/main/java/io/cattle/platform/agent/instance/factory/impl/AgentInstanceBuilderImpl.java]: @ibuildthecloud I can use system label to pass this info if you are ok with this approach

**Comment by alena1108** [comment]
@ibuildthecloud disabled *load_balancer* tables in codegen.xml

**Pull request merged**